english/sleep/marshmallow.md

```diff
@@ -37,4 +37,6 @@ Do you:
 
 [Wish for graham crackers](./crackers/crackers.md)
 
-[Sit down to meditate](sit-down-to-meditate/sit-down-to-meditate.md)
\ No newline at end of file
+[Sit down to meditate](sit-down-to-meditate/sit-down-to-meditate.md)
+
+[Wish for ice cream](./make-another-wish/ice-cream.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.